src/environments/environment.prod.ts

```diff
@@ -1,4 +1,14 @@
 // For deployment to production only.
+import {Observable} from 'rxjs/Observable';
+
+export function getServiceWorkerObservable() {
+  return new Observable(observer => {
+    setTimeout(() => {
+      observer.next();
+    }, 10000);
+  });
+}
```

Comment:
I think that it would be a better example to move this into a file like `src/app/custom-registration-strategy.ts`.